nestkernel/connector_model.h

```diff
@@ -115,57 +127,59 @@ class ConnectorModel
   bool
   is_primary() const
   {
-    return is_primary_;
+    return has_property( properties_, ConnectionModelProperties::IS_PRIMARY );
   }
 
   bool
   has_delay() const
   {
-    return has_delay_;
+    return has_property( properties_, ConnectionModelProperties::HAS_DELAY );
   }
 
   bool
   requires_symmetric() const
   {
-    return requires_symmetric_;
+    return has_property( properties_, ConnectionModelProperties::REQUIRES_SYMMETRIC );
   }
 
   bool
   requires_clopath_archiving() const
   {
-    return requires_clopath_archiving_;
+    return has_property( properties_, ConnectionModelProperties::REQUIRES_CLOPATH_ARCHIVING );
   }
 
   bool
   requires_urbanczik_archiving() const
   {
-    return requires_urbanczik_archiving_;
+    return has_property( properties_, ConnectionModelProperties::REQUIRES_URBANCZIK_ARCHIVING );
   }
 
   bool
   supports_wfr() const
   {
-    return supports_wfr_;
+    return has_property( properties_, ConnectionModelProperties::SUPPORTS_WFR );
```

Comment:
Do we still need these forwarders? Can't we just use `ConnModel.has_flag_set( ConnectionModelProperties::BLA )` from the outside? I think I would like that more, as it is more explicit and it would again remove some code here